.github/workflows/codeql-analysis.yml

```diff
@@ -13,10 +13,10 @@ name: "CodeQL"
 
 on:
     push:
-        branches: [main]
+        branches: [main, v9.x-dev]
     pull_request:
         # The branches below must be a subset of the branches above
-        branches: [main]
+        branches: [main, v9.x-dev]
```

Comment:
In the `v8.x-dev` branch, this workflow file only triggers on the `main` branch, but I guess we just forgot to update it?

https://github.com/eslint/eslint/blob/v8.x-dev/.github/workflows/codeql-analysis.yml

Reply:
> we just forgot to update it?

I believe that was the case.